### ISSUE TITLE
Include generation timestamp in json-delta endpoint

### ISSUE THREAD

**Issue by ties** [NLnetLabs/routinator#842]
As a user I want to know when a delta was generated. To do so, if would help me if this was included in the `/json-delta` API endpoint.

e.g.
```
{
  "reset": true,
  "session": "1678702338",
  "serial": 2698,
  # new attributes, same style as in `/json`
  "generated": 1679654780,
  "generatedTime": "2023-03-24T10:46:20Z",
  # ...original payload...
  "additions": [...
```

**Comment by partim** [comment]
Sounds good! I’ve added it to the 0.13 cycle.

**Comment by ties** [comment]
I do not know if this information is available for previous deltas as well. With that information you could track when the addition/removal of a entry was first "observed" by routinator

**Comment by partim** [comment]
It’s currently only available for the current version, but that is easy to change. But the ‘delta protocol’ works like RTR in the sense that you can only ask to be updated to the current version, not ask for the delta for a specific serial number. Or do I misunderstand your comment?

**Comment by partim** [comment]
Indeed. The idea was to basically do what RTR does only with JSON as the transfer format and HTTP as the transfer protocol.
